vue/components/ui/molecules/select-checkboxes/select-checkboxes.vue

```diff
@@ -14,19 +15,59 @@
     >
         <template v-slot:checkbox-group>
             <div class="checkboxes" ref="checkboxes" v-on:click.stop>
-                <checkbox-group v-bind:items="_items" v-bind:values.sync="valuesData" />
+                <search
+                    class="checkboxes-search"
+                    v-bind="{
+                        iconVisible: true,
+                        clearVisible: true,
+                        variant: 'dark',
+                        ...searchProps
+                    }"
+                    v-bind:value.sync="searchValue"
+                    v-if="search"
+                />
+                <checkbox-group v-bind:items="filteredItems" v-bind:values.sync="valuesData" />
             </div>
         </template>
     </select-ripe>
 </template>
 
 <style lang="scss" scoped>
+@import "css/variables.scss";
+
+.select-checkboxes ::v-deep .dropdown-container .dropdown > .dropdown-item.selected,
+.select-checkboxes ::v-deep .dropdown-container .dropdown > .dropdown-item.highlighted,
+.select-checkboxes ::v-deep .dropdown-container .dropdown > .dropdown-item.hover,
+.select-checkboxes ::v-deep .dropdown-container .dropdown > .dropdown-item.active,
+.select-checkboxes ::v-deep .dropdown-container .dropdown > .dropdown-item:not(.disabled):hover,
+.select-checkboxes ::v-deep .dropdown-container .dropdown > .dropdown-item:not(.disabled):active,
+.select-checkboxes ::v-deep .dropdown-container .dropdown > .dropdown-item:not(.disabled).selected,
+.select-checkboxes ::v-deep .dropdown-container .dropdown > .dropdown-item:not(.disabled).highlighted {
+    background-color: $white;

```

Comment:
let's instead create a prop to control the background color / design of the dropdown. ::v-deep should be avoided

Reply:
This isn't to customize the dropdown background, this was to fix an style issue where the dropdown color wasn't right.

If you compare the normal `select` to this `select-checkboxes` you can see that the default `background-color` doesn't match.

What is happening is that `checkbox-group` is only one item of the `dropdown`.
As it's the only item, it's always highlighted/selected, thus its color it's always the color of a `dropdown` item that is highlighted/selected istead of the actual default `$white` `background-color` of the select.

The difference without the fix:
![imagem](https://user-images.githubusercontent.com/22588915/163358832-4f421700-2e42-4d72-bd58-1f1ec5bf1efb.png)
![imagem](https://user-images.githubusercontent.com/22588915/163358865-967f4ca6-01d3-4824-bdf2-e58f42db3d4c.png)

After the fix:
![imagem](https://user-images.githubusercontent.com/22588915/163359158-f3e000c7-f602-41e9-822b-fa3a0df8a38d.png)

Reply:
OK got it. If the issue is related to the highlight of elements, let's just create a prop that enables/disables highlighting of elements and disable such value for this specific situation. This way we can avoid `v-deep` which is critical in providing good encapsulated elements that interact with each other through a properly defined interface (props).